spec/controllers/collection_items_controller_spec.rb

```diff
@@ -29,4 +99,25 @@
       end
     end
   end
+
+  describe "#destroy" do
+    before(:each) do
+      @collection = FactoryGirl.create(:collection)
+      @approved_work = FactoryGirl.create(:work)
+      @approved_work.add_to_collection(@collection) && @approved_work.save
+    end
+
+    context "destroy" do
+      let(:owner) { @collection.owners.first.user }
+      render_views
```

Comment:
I think it's left over from my attempts to get it working, and no longer needed - I'll test and see.